documentation/modules/managing/con-consumer-config-properties.adoc

```diff
@@ -267,49 +280,73 @@ If a consumer group or standalone consumer is inactive and commits no offsets du
 [source,env]
 ----
 # ...
-heartbeat.interval.ms=3000 <1>
-session.timeout.ms=45000 <2>
-auto.offset.reset=earliest <3>
+auto.offset.reset=earliest # <1>
 # ...
 ----
-<1> Adjust the heartbeat interval lower according to anticipated rebalances.
-<2> If no heartbeats are received by the Kafka broker before the timeout duration expires, the consumer is removed from the consumer group and a rebalance is initiated.
-If the broker configuration has a `group.min.session.timeout.ms` and `group.max.session.timeout.ms`, the session timeout value must be within that range.
-<3> Set to `earliest` to return to the start of a partition and avoid data loss if offsets were not committed.
+<1> Set to `earliest` to return to the start of a partition and avoid data loss if offsets were not committed.
 
 If the amount of data returned in a single fetch request is large,
 a timeout might occur before the consumer has processed it.
 In this case, you can lower `max.partition.fetch.bytes` or increase `session.timeout.ms`.
 
-== Minimizing the impact of rebalances
+== Minimizing the impact of rebalances 
 
+Rebalances in Kafka consumer groups can introduce latency and reduce throughput, impacting overall service performance. 
 The rebalancing of a partition between active consumers in a group is the time it takes for the following to take place:
 
 * Consumers to commit their offsets
 * The new consumer group to be formed
 * The group leader to assign partitions to group members
 * The consumers in the group to receive their assignments and start fetching
 
-The rebalancing process can increase the downtime of a service, particularly if it happens repeatedly during a rolling restart of a consumer group cluster.
+Rebalances are triggered by changes in consumer health, network issues, configuration updates, and scaling events.
+This process can increase service downtime, especially if it occurs frequently, such as during rolling restarts of a consumer group cluster. 
```

Comment:
We typically don't refer to a group of consumers as `consumer group cluster`. Maybe we can say something like:
`such as during rolling restarts of the consumers within the group.`